nautobot/extras/tables.py

```diff
@@ -1136,7 +1139,8 @@ class Meta(BaseTable.Meta):
 
 class AssociatedContactsTable(StatusTableMixin, RoleTableMixin, BaseTable):
     pk = ToggleColumn()
-    contact_or_team = tables.TemplateColumn(CONTACT_OR_TEAM, verbose_name="Contact/Team")
+    type = tables.TemplateColumn(CONTACT_OR_TEAM_ICON, verbose_name="Type")
```

Comment:
`type` is a python reserved keyword, Hope this is okay?

Reply:
Maybe `contact_type` instead?